### PR TITLE
Release 2020-05-22

### PR DESCRIPTION
**Pull request by FNSdev**
- Board page
- Creating ticket categories
- Creating tickets
- Updating tickets
- Assigning / unassigning members
- Responsive styles fixes

No drag-n-drop yet. Its harder than I thought :(

https://gitrello.me

### CODE REVIEW

File: gitrello/static/js/repositories/boardRepository.js [hunk 1]
```diff
@@ -30,6 +37,93 @@ class BoardRepository {
...
+                                organizationMembership: new OrganizationMembership({
+                                    id: assignment['assignee']['organization_membership']['id'],
+                                    role: assignment['assignee']['organization_membership']['role'],
+                                    user: new User({
+                                        id: assignment['assignee']['organization_membership']['user']['id'],
+                                        addedAt: assignment['assignee']['organization_membership']['user']['added_at'],
+                                        firstName: assignment['assignee']['organization_membership']['user']['first_name'],
+                                        lastName: assignment['assignee']['organization_membership']['user']['last_name'],
+                                        email: assignment['assignee']['organization_membership']['user']['email'],
+                                        username: assignment['assignee']['organization_membership']['user']['username'],
```
Comment (gmltA) [inline gitrello/static/js/repositories/boardRepository.js:86]: That looks gargantuan!

File: gitrello/static/js/repositories/boardRepository.js [hunk 2]
```diff
@@ -30,6 +37,93 @@ class BoardRepository {
...
+                                    })
+                                })
+                            })
+                        }));
+                    })
```
Comment (gmltA) [inline gitrello/static/js/repositories/boardRepository.js:91]: That looks gargantuan!

Comment (FNSdev) [reply gitrello/static/js/repositories/boardRepository.js:91]: :(

**Comment by FNSdev** [comment]
> Looks great.
> What else is on the horizon?

@gmltA 
I hope that`s enough 😃 

I`ll try to make it possible to drag`n`drop tickets at least in the same category.

I don`t think that I have enough time to add decent Github Integration, but I can try to add some minor features (creating branches for tickets or smth like that).

Or I can add comments section for tickets.

**Comment by gmltA** [comment]
I'd be pretty satisfied with d'n'd functionality only. You think you'll manage to do that till exam?

**Comment by FNSdev** [comment]
I will try to.

**Comment by gmltA** [comment]
> I will try to.

Fine. Maybe not 100% production-ready, but some initial implementation would be a great thing to look through.

**Pull request merged**